### PR TITLE
[RU] 01-2 translation

### PR DESCRIPTION
**Pull request by MrPark97**

### CODE REVIEW

File: docs/ru/week01/01-2.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
+---
+lang-ref: ch.01-2
+lecturer: Yann LeCun
+title: Evolution and Uses of CNNs and Why Deep Learning?
```
Comment (Atcold) [inline docs/ru/week01/01-2.md:4]: Title needs translation.

**Review by Atcold — CHANGES_REQUESTED** [review]
Good job, thanks!
Please, check comment below.

**Comment by Atcold** [comment]
@vladisai, can you review?

File: docs/ru/week01/01-2.md (hunk 1 above)
Comment (MrPark97) [reply docs/ru/week01/01-2.md]: fixed

**Review by Atcold — APPROVED** [review]
Looks good to me. Merging. @vladisai can review what's merged.

**Pull request merged**